lumigator/frontend/src/types/Experiment.ts

```diff
@@ -0,0 +1,134 @@
+export type Experiment = {
+  id: string;
+  created: string;
+  dataset: unknown;
+  description: string;
+  name: string;
+  experimentStart: string;
+  jobs: Job[];
+  useCase: string;
+  runTime: string;
+  samples?: number;
+  models: Model[];
+  max_samples?: number;
+  status: string;
+};
```

Comment:
These types are likely neither accurate nor comprehensive, but its a good start to have some typechecking and know what the data structure looks like